modules/how-etcd-works.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+// * etcd/etcd-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="how-etcd-works_{context}"]
+= How etcd works
+
+To ensure a reliable approach to cluster configuration and management, etcd uses the etcd Operator. The Operator simplifies the use of etcd on a Kubernetes container platform such as {product-title}. With the etcd Operator, you can complete the following tasks: 
+
+* Create or delete etcd members 
+* Resize clusters 
+* Perform backups
+* Upgrade etcd 
```

Comment:
Is this coming from existing content somewhere?

While this is not inaccurate, it's not fully correct to say that the openshift cluster etcd operator let's an admin perform these operations directly via the operator's etcd API.

E.g create/delete of members or resizing of nodes is done via the machine API or by manually deleting nodes/members.
Upgrades of the operator and operand are also handled via the CVO and not the etcd-operator directly.

Backups are the only operation that are directly handled by the etcd-operator but that's also in tech preview atm.

So the etcd-operator is facilitating all of these operations but just want to be clear that most of them are not directly initiated or handled by the operator.